openmdao/solvers/linesearch/tests/test_backtracking.py

```diff
@@ -443,6 +443,7 @@ def test_linesearch_vector_bound_enforcement(self):
             assert_near_equal(top['comp.z'][ind], [1.5], 1e-8)
 
         msg = (f"'comp.z' exceeds lower bounds\n  Val: [1.33333333 1.33333333 1.33333333]\n  Lower: [1.5 1.5 1.5]\n")
+        top.run_model()
```

Comment:
Was this run_model call just leftover from debugging or was it intentional?

Reply:
resolved